packages/main/test/samples/Label.sample.html

```diff
@@ -60,48 +60,40 @@ <h3>Wrapped Label</h3>
 	<h3>Label 'for'</h3>
 	<div class="snippet" style="display: flex;flex-direction: column;">
 		<ui5-label id="myLabel" for="myInput" required show-colon>First name</ui5-label>
-		<ui5-input id="myInput" aria-required="true" accessible-name-ref="myLabel" placeholder="Enter your name"></ui5-input>
+		<ui5-input id="myInput" required accessible-name-ref="myLabel" placeholder="Enter your name"></ui5-input>
 		<br>
 		<ui5-label id="myLabel2" for="myDP" required show-colon>Date of birth</ui5-label>
-		<ui5-date-picker id="myDP" accessible-name-ref="myLabel2" aria-required="true"></ui5-date-picker>
+		<ui5-date-picker id="myDP" accessible-name-ref="myLabel2" required></ui5-date-picker>
 		<br>
 		<ui5-label id="myLabel3" for="mySelect" required show-colon>Job</ui5-label>
-		<ui5-select id="mySelect" aria-required="true" accessible-name-ref="myLabel3">
+		<ui5-select id="mySelect" required accessible-name-ref="myLabel3">
```

Comment:
Are all those accessible-name-ref attributes still needed - since now the for attribute should do the job?